lib/validators/liveValidator.ts

```diff
@@ -372,11 +372,20 @@ export class LiveValidator {
     }
   }
   private toLiveValidationIssue(err: { [index: string]: any; url: string }): LiveValidationIssue {
```

Comment:
these errors can happen also in model validation, i would suggest is to apply this for that also

Reply:
we might be able to centralize that when the error is created in Sway

Reply:
I checked the code and the path is built from the error and its inner error recursively.
So I don't think modifying path could be done easily in Sway without regression. It is much safer to handle it on the top level.

Reply:
yes, as we checked yesterday did you try centralizing it in `setPathProperties` properties. that way we we minimize the changes and they will apply to model validation/ semantic validation and jsonpath and jsonpointer